fl4health/model_bases/moon_base.py

```diff
@@ -14,10 +14,11 @@ def __init__(
         self.head_module = head_module
 
     def forward(self, input: torch.Tensor) -> Dict[str, torch.Tensor]:
+        # input is expected to be of shape (batch_size, *)
         x = self.base_module.forward(input)
         if self.projection_module:
```

Comment:
I know you didn't change this code, but wanted to just suggest a "pythonic" collapsing of the if-else. That is
```
p = self.projection_module.forward(x) if self.projection_module else x
```